metrics/github/security.py

```diff
@@ -76,6 +76,14 @@ def is_closed_at(self, target_date):
             self.dismissed_at is not None and self.dismissed_at <= target_date
         )
 
+    @staticmethod
+    def from_dict(my_dict):
+        return Vulnerability(
+            dates.date_from_iso(my_dict["createdAt"]),
+            dates.date_from_iso(my_dict["fixedAt"]),
+            dates.date_from_iso(my_dict["dismissedAt"]),
+        )
```

Comment:
To avoid repeating `dates.date_from_iso`

```suggestion
        fields = [my_dict["createdAt"], my_dict["fixedAt"], my_dict["dismissedAt"]]
        return Vulnerability(*map(dates.date_from_iso, fields))
```

Reply:
That's more succinct, but I'm not sure it's more readable, although I can see the argument either way. I think I'll leave it as-is for the moment though.

Reply:
The as-is approach would make it easier for `Vulnerability` to take a non-date property, too.